CorsixTH/Src/th_gfx.cpp

```diff
@@ -1298,7 +1299,7 @@ bool THAnimation_is_multiple_frame_animation(drawable* pSelf) {
 animation_base::animation_base() : drawable() {
   x_relative_to_tile = 0;
   y_relative_to_tile = 0;
-  for (int i = 0; i < 13; ++i) {
+  for (int i = 0; i < MAX_NUMBER_OF_LAYERS; ++i) {
```

Comment:
We only use screaming snake case for macros per: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-all-caps

Reply:
What do you suggest to use ?

Reply:
Going by the lower comment of constexpr I would say normal snake case...